code/renderers/vue/package.json

```diff
@@ -77,6 +79,9 @@
   "engines": {
     "node": ">=16.0.0"
   },
+  "installConfig": {
+    "hoistingLimits": "workspaces"
```

Comment:
I needed to add this to keep `vue-template-compiler` from being hoisted up to the monorepo root, where it would find vue3 (also hoisted) and crash.

Reply:
We're removing Vue2 in SB 8 anyway, so this doesn't matter.